P/Powsybl/build_tarballs.jl

```diff
@@ -0,0 +1,86 @@
+using BinaryBuilder, Pkg
+
+name = "Powsybl"
+version = v"0.1.0"
+
+# See https://github.com/JuliaLang/Pkg.jl/issues/2942
+# Once this Pkg issue is resolved, this must be removed
+uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
+delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
+
+pypowsybl_version = v"1.7.0"
+
+
+sources = [
+    GitSource("https://github.com/powsybl/powsybl.jl.git", "0a9e3110c366808e7ac23cfd9e19f51793807392"),
+    GitSource("https://github.com/powsybl/pypowsybl.git", "cd5fea41bbfb2897fd71a6e63b2d07a465055699"),
+    ArchiveSource("https://github.com/powsybl/pypowsybl/releases/download/v$(pypowsybl_version)/binaries-v$(pypowsybl_version)-windows.zip",
+                  "82d3cee44992dcceaee7549f17351155e91c9eb2bdce97b1cf6c0107155991e8",
+                  "powsybl-java-x86_64-w64-mingw32"),
+    ArchiveSource("https://github.com/powsybl/pypowsybl/releases/download/v$(pypowsybl_version)/binaries-v$(pypowsybl_version)-linux.zip",
+                  "8832e1ff432e97807dc6dfddb4b001dd2c3c05a7411fc3748c8af3854a3b448c",
+                  "powsybl-java-x86_64-linux-gnu"),
+    ArchiveSource("https://github.com/powsybl/pypowsybl/releases/download/v$(pypowsybl_version)/binaries-v$(pypowsybl_version)-darwin.zip",
+                  "d541eb07a334d9272b167cb30f7d846ff109db49ac61a9776593c1aface18324",
+                  "powsybl-java-x86_64-apple-darwin")
```

Comment:
Ugh, didn't notice you're targeting also musl, with my suggestion you need to duplicate the source (at least it should be downloaded only once)
```suggestion
    ArchiveSource("https://github.com/powsybl/pypowsybl/releases/download/v$(pypowsybl_version)/binaries-v$(pypowsybl_version)-linux.zip",
                  "8832e1ff432e97807dc6dfddb4b001dd2c3c05a7411fc3748c8af3854a3b448c",
                  "powsybl-java-x86_64-linux-gnu"),
    # We need to duplicate this source for musl
    ArchiveSource("https://github.com/powsybl/pypowsybl/releases/download/v$(pypowsybl_version)/binaries-v$(pypowsybl_version)-linux.zip",
                  "8832e1ff432e97807dc6dfddb4b001dd2c3c05a7411fc3748c8af3854a3b448c",
                  "powsybl-java-x86_64-linux-musl"),
    ArchiveSource("https://github.com/powsybl/pypowsybl/releases/download/v$(pypowsybl_version)/binaries-v$(pypowsybl_version)-darwin.zip",
                  "d541eb07a334d9272b167cb30f7d846ff109db49ac61a9776593c1aface18324",
                  "powsybl-java-x86_64-apple-darwin14"),
```